package.json

```diff
@@ -22,7 +22,6 @@
     "uglify-js": "1.3.3"
   },
   "devDependencies": {
-    "cli-table": "0.2.0",
```

Comment:
No I checked and it's not so I removed it

On 13 August 2015 at 09:42, Adam Silver <notifications@github.com> wrote:

> In package.json
> <https://github.com/cinsoft/jessie/pull/309#discussion_r36951885>:
>
> > @@ -22,7 +22,6 @@
> >      "uglify-js": "1.3.3"
> >    },
> >    "devDependencies": {
> > -    "cli-table": "0.2.0",
>
> I think this is used in CLI builder?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cinsoft/jessie/pull/309/files#r36951885>.
>